subworkflows/local/bam_variant_calling_mpileup/main.nf

```diff
@@ -1,17 +1,24 @@
 include { CAT_CAT as CAT_MPILEUP         } from '../../../modules/nf-core/cat/cat/main'
-include { SAMTOOLS_MPILEUP               } from '../../../modules/nf-core/samtools/mpileup/main'
+include { BCFTOOLS_MPILEUP               } from '../../../modules/nf-core/bcftools/mpileup/main'
+include { GATK4_MERGEVCFS                } from '../../../modules/nf-core/gatk4/mergevcfs/main'
+
 
 workflow BAM_VARIANT_CALLING_MPILEUP {
     take:
         cram                    // channel: [mandatory] [meta, cram, interval]
         fasta                   // channel: [mandatory]
+        dict
 
     main:
 
     ch_versions = Channel.empty()
 
-    SAMTOOLS_MPILEUP(cram, fasta)
-    mpileup = SAMTOOLS_MPILEUP.out.mpileup.branch{
+    BCFTOOLS_MPILEUP(cram, fasta,true)
```

Comment:
Can you comment what is this `true` value for?
like we do for this subworkflow:
https://github.com/nf-core/sarek/blob/d854ccc64ca6e14366bddd5a771d7b25b96f61c3/workflows/sarek.nf#L494-L495